proto/ibc/core/channel/v1/channel.proto

```diff
@@ -65,6 +65,12 @@ enum State {
   // A channel has been closed and can no longer be used to send or receive
   // packets.
   STATE_CLOSED = 4 [(gogoproto.enumvalue_customname) = "CLOSED"];
+  // A channel has just started the channel upgrade handshake.
+  // The chain that is proposing the upgrade should set the channel state from OPEN to UPGRADEINIT.
+  STATE_UPGRADEINIT = 5 [(gogoproto.enumvalue_customname) = "UPGRADEINIT"];
```

Comment:
I think we should go for `INITUPGRADE` and `TRYUPGRADE` to align with the `TRYOPEN` state.
Apologies for the nit picking 😆

Reply:
ah you're right makes total sense!